src/components/Proposal/ProfileProposalItem.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { Link } from 'decentraland-gatsby/dist/plugins/intl'
+import Time from 'decentraland-gatsby/dist/utils/date/Time'
+import { Card } from 'decentraland-ui/dist/components/Card/Card'
+import { Header } from 'decentraland-ui/dist/components/Header/Header'
+import { Mobile, NotMobile } from 'decentraland-ui/dist/components/Media/Media'
+
+import { VotedProposal } from '../../entities/Votes/types'
+import locations from '../../modules/locations'
+import CategoryPill from '../Category/CategoryPill'
+import ChevronRightCircleOutline from '../Icon/ChevronRightCircleOutline'
+import StatusPill from '../Status/StatusPill'
+
+import './ProfileProposalItem.css'
+import ResultStance from './ResultStance'
+
+interface Props {
+  votedProposal: VotedProposal
+}
+
+function ProfileProposalItem({ votedProposal }: Props) {
+  const t = useFormatMessage()
+  const { proposal } = votedProposal
+  const choiceIdx = votedProposal.choice - 1
+  const { choices, finish_at, scores } = proposal
+  const dateText = t(`page.home.open_proposals.${Time().isBefore(Time(finish_at)) ? 'ends_date' : 'ended_date'}`, {
+    value: Time(finish_at).fromNow(),
+  })
+  const maxScore = Math.max(...scores)
+  const resultMatches = maxScore === scores[choiceIdx]
```

Comment:
I don't get this  👀 
You are checking if the vp of the voted choice is the same as the winning score?
Could you find the winner choice and match that against the selected choice instead? 
(It makes more sense to me to compare by choices instead of by score)